app/frontend/src/aquifers/components/View.vue

```diff
@@ -13,7 +13,7 @@
 */
 
 <template>
-  <b-card class="container p-1">
+  <b-card class="container container-wide p-0 card-container pb-3" :class="{ 'p-4': editMode }">
```

Comment:
can you double check this works as intended (second `class` attribute)?

Second thing is that I think the container-wide class has been removed in our code base.  `container` now has an extra media query for 1080p screens with a max width of 1400px (App.vue).

Reply:
yes it does work as intended.